### DISCUSSION TITLE
Blitzwolf BW-SHP15 no power, no current and no voltage

### DISCUSSION THREAD

**Discussion by manustars** [Koenkk/zigbee2mqtt#15020] [Devices]
hi to all, i have 4 BW-SHP15 and 3 works fine but 1 i no see the value on object 
![immagine](https://user-images.githubusercontent.com/60410607/202158702-b2ebede7-7899-4c18-8d4d-7a00273897a4.png)

**Comment by fliespl** [comment]
Had the same with tuya (1 out of 4 was reporting 0 instead of n/a). One of them was simply broken :)

**Comment by riegelbrau** [comment]
Not exactly the same issue, but no reporting of the actual current and power with a new BlitzWolf BW-SHP15. I did not change anything with the unit, only plugged it in, pressed the button for more than 5 seconds, and it appeared in my Z2M network.
![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/73437730/4a345cdc-a26a-440e-a86f-9aebfd79000e)
These are the details:
![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/73437730/e33486c7-b3b6-48e0-94ec-d23fc7cb4c96)
For polling handling, I entered the 5 in this field:
![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/73437730/d82183c4-64a4-4a41-aaf2-ebd80b77ba2c)
Nothing appears when using the plug with some power consuming machine. Do I need any action to initiate polling?

Regards Christoph

**Comment by riegelbrau** [comment]
I just solved it myself! In a different issue I found some information about setting the precision level for the values. So I entered into the Specific Settings section and clicked into the power_precision setting and all the other precision related ones. 
![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/73437730/5aedcda0-6ed8-401c-ab6d-494cbfdf3320)
With the next reporting the power and current values were displayed!
![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/73437730/fd69d8de-6c84-4ee4-964c-3d289ee67f2f)
I assume that these settings are not properly initiated by the software. Only after changing the setting, it is accepted as a correct setting.

The UI is some kind of unintuitive. Why do we need sliders for one-digit number values and how should we submit "ENTER"? I tried clicking into a different field, but somtimes the  settings were not completely saved. 

It looks as if the precision for Voltage and Power is useless. Maybe the unit does only send the data without decimal digits. If that's true, the settings could be removed.

Regarding the measurement_poll_interval it looks as if this is not really valuable. From my experience the interval is changing and is much slower then my setting of 2  or 3 seconds. 

Regards Christoph
